docs/migrating-to-v6.md

```diff
@@ -608,26 +592,9 @@ Here are all the specific changes that we made to the `Utils` object:
 
    </div>
 
-1. `Shopify.Utils.deleteCurrentSession` is now `shopify.session.deleteCurrent`, it takes in an object, and the `isOnline` param is mandatory.
-   <div>Before
-
-   ```ts
-   const session = await Shopify.Utils.deleteCurrentSession(req, res);
-   ```
-
-   </div><div>:warning: After
+1. `Shopify.Utils.deleteCurrentSession` has been removed, as the library no longer handles the storage of sessions.
```

Comment:
What is the best practice / recommendation for when an app should delete a session?

Reply:
The main event to trigger deletion would be when the app is uninstalled by the shop (e.g., in response to `APP_UNINSTALLED` webhook).